processing/data_collection/gazette/settings.py

```diff
@@ -10,3 +10,13 @@
 }
 FILES_STORE = "/mnt/data/"
 QUERIDODIARIO_EXTRACT_TEXT_FROM_FILE = True
+
+EXTENSIONS = {
+    "spidermon.contrib.scrapy.extensions.Spidermon": 500,
+}
+SPIDERMON_ENABLED = True
+ITEM_PIPELINES = {
```

Comment:
**issue (blocking)**: We already have a list of ITEM_PIPELINES defined. This change will disable all other pipelines.